nix/tools.nix

```
@@ -85,6 +85,7 @@
 , go-tools
 , golangci-lint
 , golines
+, gotools
```

We need to make sure we can evaluate this on older nixpkgs, as this tool is quite new to nixpkgs.

```suggestion
, gotools ? null
```